src/components/shared/Tooltip.js

```diff
@@ -82,25 +82,43 @@ export default class Tooltip extends React.PureComponent<Props, State> {
     const { children, mouseX, mouseY } = this.props;
     const { interiorElement } = this.state;
 
-    const offsetX = interiorElement
-      ? Math.max(0, mouseX + interiorElement.offsetWidth - window.innerWidth)
-      : 0;
+    // By default, position the tooltip below and at the right of the mouse cursor.
+    let top = mouseY + MOUSE_OFFSET;
+    let left = mouseX + MOUSE_OFFSET;
 
-    let offsetY = 0;
     if (interiorElement) {
       if (
-        mouseY + interiorElement.offsetHeight + MOUSE_OFFSET >
+        mouseY + MOUSE_OFFSET + interiorElement.offsetHeight >=
         window.innerHeight
       ) {
-        offsetY = interiorElement.offsetHeight + MOUSE_OFFSET;
-      } else {
-        offsetY = -MOUSE_OFFSET;
+        // if the tooltip doesn't fit below the mouse cursor
```

Comment:
I'm finding these comments difficult to reason with. I think it would be better to write complete sentences here. I think also starting the comment with the word `if` makes me confused about what the code is doing. Generally I find it easier if the interior of the `if` block states what the condition is inside that block, and above the `if` to describe the conditions being checked. Since this statement is nested between two `if` statements, I don't know what it's referring to.

nit: Also, please use proper capitalization and punctuation.